Task3.1/3.1.2/TextDictionary/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TextDictionary
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+                Console.Write("Введите текст:");
+                string text = Console.ReadLine().ToUpper();
+                char[] chars = new char[] { ' ', ',', '.', '?', '!', '-', '"' };
+                string[] result = text.Split(chars, StringSplitOptions.RemoveEmptyEntries);
+                Dictionary<string, int> textDictionary = new Dictionary<string, int>();
+                textDictionary.FirstOrDefault(res => res.Key == result[0]);
+                for (int obj = 0; obj < result.Length; obj++)
+                {
+                    if (textDictionary.ContainsKey(result[obj]))
+                    {
+                        textDictionary[result[obj]]++;
+                    } else
+                    {
+                        textDictionary.Add(result[obj], 1);
+                    }
+                }
+                foreach (var element in textDictionary)
+                {
+                    Console.WriteLine($"Данное слово {element.Key} встречается {element.Value} раз.");
+
+                }
+                if (textDictionary.Where(el => el.Value > 3).Count() < textDictionary.Count() * 0.2)
```

Comment:
```suggestion
                if (textDictionary.Count(el => el.Value > 3) < textDictionary.Count * 0.2)

```
Когда есть возможность, лучше пользоваться свойством Count, потому что метод Count() будет делать проход по все коллекции для подсчета количества